### ISSUE TITLE
Patterns are mirrored along y axis in pdf output

### ISSUE THREAD

**Issue by igidrau** [typst/typst#2846]
### Description

When compiling the following document:
```typst
#set page(width: auto, height: auto, margin: 0pt)

#let pat = pattern(size: (10pt, 10pt), line(stroke: 4pt, start: (0%, 0%), end: (100%, 100%)))
#rect(width: 50pt, height: 50pt, fill: pat)
```
The output document is different depending on the target:
| svg / png | pdf |
| ----- | ---- |
| ![test.png](https://github.com/typst/typst/assets/36123998/0014c66a-b667-4fdf-951b-2fb4f9377a3a) | ![test pdf](https://github.com/typst/typst/assets/36123998/e8edd155-051f-45ab-99d9-01574db02aff) |

It looks like the pdf output is drawn bottom to top instead of top to bottom like the png and svg outputs and normal content.
(@Dherse)

### Operating system

Linux

### Typst version

- [X] I am using the latest version of Typst

**Comment by Dherse** [comment]
This is known, there is also an offset, it will be fixed before 0.10.0 is officially released.

**Comment by Dherse** [comment]
Ok, it's already fixed, just need to open up a PR
